Snakefile

```diff
@@ -55,47 +64,60 @@ rule all:
         ],
 
 
-def devtools_install():
-    if config["ploidetect_local_clone"] and config["ploidetect_local_clone"] != "None":
-        install_path = config["ploidetect_local_clone"].format(**config)
-        devtools_cmd = f"devtools::install_local('{install_path}', force = TRUE)"
-    else:
-        ver = config["ploidetect_ver"]
-        devtools_cmd = f"devtools::install_github('lculibrk/Ploidetect', ref = '{ver}')"
-    return f'"{devtools_cmd}"'
-
 rule download_cytobands:
     """Downloads cytoband data for plotting & (todo) hgver-specific centromere filtering"""
     input:
         HTTP.remote(
-            expand("http://hgdownload.cse.ucsc.edu/goldenpath/{hgver}/database/cytoBand.txt.gz", hgver = config["genome_name"])
-        )
+            expand(
+                "http://hgdownload.cse.ucsc.edu/goldenpath/{hgver}/database/cytoBand.txt.gz",
+                hgver=config["genome_name"],
+            )
+        ),
     output:
-        expand("resources/{hgver}/cytobands.txt", hgver = config["genome_name"])
+        expand("resources/{hgver}/cytobands.txt", hgver=config["genome_name"]),
     resources:
         cpus=1,
         mem_mb=MEM_PER_CPU,
+    conda:
+        "conda_configs/sequence_processing.yaml"
     shell:
         "gunzip -c {input} > {output}"
 
+
+ploidetect_install_cmd = (
+    "devtools::install_github('lculibrk/Ploidetect', ref = '{}')".format(
+        config["ploidetect_ver"]
+    )
+)
+if config["ploidetect_local_clone"] and config["ploidetect_local_clone"] != "None":
+    install_path = config["ploidetect_local_clone"].format(**config)
+    ploidetect_install_cmd = f"devtools::install_local('{install_path}', force = TRUE)"
+ploidetect_install_cmd = f'"{ploidetect_install_cmd}"'
```

Comment:
This removes the def devtools_install() function.

Mostly change just because snakemake --lint suggests separating functions from rules.